app/main.py

```diff
@@ -1,3 +1,21 @@
+import types
+from typing import Optional
+import os
+
+
 class CleanUpFile:
-    # write your code here
-    pass
+    def __init__(self, filename: str) -> None:
+        self.filename = filename
+
+    def __enter__(self) -> object:
+        return self
+
+    def __exit__(
+        self,
+        exc_type: Optional[type[BaseException]],
+        exc_value: Optional[BaseException],
+        traceback: Optional[types.TracebackType],
+    ) -> bool:
+        if os.path.exists(self.filename):
+            os.remove(self.filename)
+        return False
```

Comment:
Do you need to return False here?

Reply:
I guess not) Since (according to gpt) "if a context manager's __exit__ method does not explicitly return True, Python will treat it as though it returned False by default"